language/documentation/book/src/integers.md

```diff
@@ -21,23 +23,29 @@ If a literal is too large for its specified (or inferred) size range, an error i
 let explicit_u8 = 1u8;
 let explicit_u64 = 2u64;
 let explicit_u128 = 3u128;
+let explicit_u8_underscored = 1_2_3u8;
+let explicit_u64_underscored = 24_5_u64;
+let explicit_u128_underscored = 3000_0__000_1u128;
 
 // literals with simple inference
 let simple_u8: u8 = 1;
 let simple_u64: u64 = 2;
 let simple_u128: u128 = 3;
+let simple_u8_underscored: u8 = 1_1_;
```

Comment:
I don't think we need extra examples in inference--different literal representations don't change inference behavior.